src/jwplayer/utils/articleVideo/articleVideoConfig.ts

```diff
@@ -26,13 +26,43 @@ const getNormalizedPlaylistIndex = (playlist) => {
 	return playerImpressions > playlist.length ? playerImpressions % playlist.length : playerImpressions;
 };
 
+const getAdvertising = (lang: string) => {
```

Comment:
nit-pick: consider renaming to `getAdvertisingConfig()` or `getAdvertisingSetup()` ;)